### ISSUE TITLE
Clean up themes

### ISSUE THREAD

**Issue by jleclanche** [lxde/lxqt#327]
- Theme review
- Import Numix
- Get rid of old dark theme, merge into dark alpha
- anything else?

**Comment by jleclanche** [comment]
@intialonso What's the status of all this? What's left to be done before release?

**Comment by intialonso** [comment]
I think that all the themes are in good shape, just some code cleaning are
left.

The Light theme was left as original (a Razor memory), just added the new
features.

I missed more testing and bug reports.

2015-01-18 6:21 GMT-05:00 Jerome Leclanche <notifications@github.com>:

**Comment by paulolieuthier** [comment]
@intialonso you can remove `lxqt-power.qss` and `lxqt-lightdm-greeter.qss`, those files are deprecated now. Besides that, the only thing missing here is the theme naming suggested by you. I like the .desktop file approarch (maybe others dislike it, though). What would it require? A name, a description, an icon, an author and wheter it requires a compositor? We should do that and try to make it similar to you mock up (which is really nice, btw). Maybe it could be done before 0.9?

**Comment by intialonso** [comment]
I will do the clean this night at Home. I agree with your.desktop proposal.

**Comment by intialonso** [comment]
Do we have the support ready for the .desktop files in lxqt-config-appearance?

If yes, the thumbnail must be the icon line in the .desktop file?

For example, is this right?:

```
[Desktop Entry]
Type=Theme
Version=0.9
Name=Frost
Comment=A cold flat theme with blue highlights. Does not require composition.
Auhor=LXQT Team
Icon=preview.png
```

**Comment by jleclanche** [comment]
Not for 0.9.

**Comment by jleclanche** [comment]
Alright guys, most of this is done, moving it off the 0.9 milestone as we still have some cleanups left to do in 1.0. @intialonso is there anything left to do before the release? Can you do a quick theme review while we're still in freeze? I'm still targeting a release on the 29th.

**Comment by intialonso** [comment]
I think that this is ready for release, there is no big bugs or problems
here (at least not reported). I will review all again and will do a little
bug hunt.



2015-01-26 20:42 GMT-05:00 Jerome Leclanche <notifications@github.com>:

**Comment by paulolieuthier** [comment]
We should at least capitalize the themes' names for 0.9.

**Comment by jleclanche** [comment]
Sure. Want to do it?

**Comment by paulolieuthier** [comment]
Done here: ad1f6f9713ab0b0939999f7b2e40f41c6c6f6cda. Please test it.

**Comment by jleclanche** [comment]
We'll need to make sure that's gone next release; the names need to be translatable so we cant just get them from the folder.

**Comment by paulolieuthier** [comment]
Yes, please create an issue for that so we won't forget it.

**Comment by paulolieuthier** [comment]
I reverted last commit because it caused too much trouble. No capitalization on themes' names.

**Comment by paulolieuthier** [comment]
Capitalization landed in lxde/lxqt-config@c05e5809cd1b18b27e4f7f139d1ef7fb939b3728. No problems this time.

**Comment by paulolieuthier** [comment]
@intialonso ping. If you think themes are ready for this release, please close this issue.
